homeassistant/components/netgear/strings.json

```
@@ -0,0 +1,31 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "title": "Netgear",
+        "description": "Default host: {host}\n Default port: {port}\n Default username: {username}",
+        "data": {
+          "host": "Host (Optional)",
+          "port": "Port (Optional)",
+          "ssl": "Use SSL (Optional)",
+          "username": "Username (Optional)",
+          "password": "Password"
```

Use key references;
A boolean is always optional: `Use SSL (Optional)` -> `Uses an SSL certificate`.
```suggestion
          "host": "[%key:common::config_flow::data::host%] (Optional)",
          "port": "[%key:common::config_flow::data::port%] (Optional)",
          "ssl": "[%key:common::config_flow::data::ssl%]",
          "username": "[%key:common::config_flow::data::username%] (Optional)",
          "password": "[%key:common::config_flow::data::password%]"
```